aiida/cmdline/params/options/commands/code.py

```diff
@@ -50,7 +53,10 @@ def is_not_on_computer(ctx):
     prompt_fn=is_not_on_computer,
     type=click.Path(file_okay=False, exists=True, readable=True),
     cls=InteractiveOption,
-    help=('[if --store-in-db]: directory containing the executable and all other files necessary for running it.')
+    help=(
+        '[if --store-in-db]: Absolute path to directory containing the executable and all other files necessary for running it'
```

Comment:
Yep sounds reasonable to me, shall we open an issue about this then?

Reply:
I agree the "DB" is technically not correct here.

Let's open an issue for this - this choice of name is a bit delicate since there has been quite some confusion about local/remote codes in the past, so any suggestion should be tested on an AiiDA novice ;-)

Reply:
see #4224